tonic-health/Cargo.toml

```diff
@@ -21,11 +19,16 @@ transport = []
 [dependencies]
 async-stream = "0.3"
 prost = "0.13"
-tokio = {version = "1.0", features = ["sync"]}
+tokio = { version = "1.0", features = ["sync"] }
 tokio-stream = "0.1"
-tonic = { version = "0.12", path = "../tonic", default-features = false, features = ["codegen", "prost"] }
+tonic = { 
```

Comment:
On the other hand, not a big fan of this. IMO keeping it on one line makes it the list of dependencies easier to comprehend.